.github/CODEOWNERS

```diff
@@ -0,0 +1,39 @@
+# This does not cover all the code in edx-platform but it's a good start.
```

Comment:
Other folders we called out based on running the following:
```
git log --pretty=format: --name-only --since "1 year ago" | grep -ve ".po$" | grep -ve ".mo$" | grep -ve ".sql$" | grep -ve ".json$" | grep -ve ".txt$" | sort | uniq -c | sort -rg | head -30
```

* lms/djangoapps/instructor_task/
* openedx/features/content_type_gating/
* openedx/features/course_duration_limits/

Reply:
Added.